ts/src/test/Exchange/base/test.order.ts

```diff
@@ -45,9 +45,10 @@ function testOrder (exchange: Exchange, skippedProperties: object, method: strin
     testSharedMethods.assertGreaterOrEqual (exchange, skippedProperties, method, entry, 'amount', exchange.safeString (entry, 'remaining'));
     testSharedMethods.assertGreaterOrEqual (exchange, skippedProperties, method, entry, 'amount', exchange.safeString (entry, 'filled'));
     if (!('trades' in skippedProperties)) {
+        const skippedNew = exchange.deepExtend (skippedProperties, { 'timestamp': true, 'datetime': true, 'side': true });
```

Comment:
specifically in `test.order`, the trades should have those keys skipped. in future, i'll try to revise this and might remove `side` from skipped, but these skips were needed for now.